.github/pull_request_template.md

```diff
@@ -1,12 +1,49 @@
+# Thank you for submitting a pull request to newrelic/developer-website
+
 ## Description
+
 Add a short description about the purpose of this PR here.
 
 ## Reviewer Notes
+
 If there are links or steps needed to test this work, add them here.
 
 ## Related Issue(s) / Ticket(s)
+
 If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
 * [issue]()
 
 ## Screenshot(s)
+
 If relevant, add screenshots here.
+
```

Comment:
At the point of creating a PR, most of the commits will have already been pushed (and therefore can not be updated easily). We should probably put this content elsewehere.

Reply:
Oh hah..I see it in `CONTRIBUTING.md` 👌

Reply:
We still may want to remove this from the `pull_request_template.md`

Reply:
@zstix Yeah, it's interesting as OSS has it set in the PR template, if you are at this point and you see it you could fix your commit statement with: 

```
git commit --amend -m "New message" 
git push --force repository-name branch-name
```
I've done this on many occasions when I saw the PR template reminder. :) 

but if you already submitted the PR, you'd have to do  rebase interactive to combine or modify commits

git rebase -i HEAD~

https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

Reply:
The problem with this approach is that, if someone forks or branches off of a commit that gets rebased, they will be in a detached state and are forced to rebased themselves. It's a bit in the minutiae of git workflows tho...this is probably good

Reply:
@zstix so is it cool to leave this in the PR template? I'm happy to take it out if it won't really help. I just OSS had their template this way, so I thought it may be useful.

Reply:
Let's leave it in and see how it feels. Thanks for working through it with me :+1: